amplify/backend/ts-code/src/api/AddItem.ts

```diff
@@ -10,6 +10,8 @@ import { emitAPIMetrics } from "../metrics/MetricsHelper"
 /**
  * Adds item to item inventory table.
  */
+const uniqueId = Math.floor(Math.random() * Date.now()).toString().substring(0,5)
```

Comment:
Can you make it into a method inside of the AddItem class? I feel like that's cleaner. Something like
```
private getUniqueId(): string
```